src/Symfony/Component/Validator/Tests/Constraints/JsonValidatorTest.php

```diff
@@ -0,0 +1,74 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Validator\Tests\Constraints;
+
+use Symfony\Component\Validator\Constraints\Json;
+use Symfony\Component\Validator\Constraints\JsonValidator;
+use Symfony\Component\Validator\Test\ConstraintValidatorTestCase;
+
+class JsonValidatorTest extends ConstraintValidatorTestCase
+{
+    protected function createValidator()
+    {
+        return new JsonValidator();
+    }
+
+    public function testJsonIsValid()
+    {
+        $this->validator->validate(
```

Comment:
still.. i think we should assert some non-complex JSON values as well, e.g.

- `null`
- `1`
- `true`
- `"string"`

Reply:
yes, there are two other assertions with, `testNullIsValid()` and `testEmptyStringIsValid` , i'm going to add the other cases in  `getInvalidValues`.

Reply:
there's a different bewteen PHP's `null` and JSON string valuie `"null"`